.circleci/config.yml

```diff
@@ -14,6 +14,15 @@ python35_image: &python35_image circleci/python:3.5
 python27_image: &python27_image circleci/python:2.7
 ddtrace_dev_image: &ddtrace_dev_image datadog/dd-trace-py:latest
 datadog_agent_image: &datadog_agent_image datadog/docker-dd-agent:latest
+test_agent: &test_agent
+  - image: kyleverhoog/dd-trace-test-agent:latest
```

Comment:
note to self: pin this in case for consistent builds

Reply:
and please use a Dockerfile from this repo or a repo where it's clear we maintain/can modify.

Reply:
:+1: definitely! This will be the case once the repo's public 🙂